models/models/cohort.py

```diff
@@ -0,0 +1,97 @@
+import json
+
+from pydantic import BaseModel
+
+from models.base import SMBase
+from models.models.sequencing_group import SequencingGroup, SequencingGroupExternalId
+
+
+class Cohort(SMBase):
+    """Model for Cohort"""
+
+    id: str
```

Comment:
Is this intended to be a `str` like `COH17` or the string form of an `int` like `1`?

Reply:
Oops! This should be an int! have adjusted.